src/workflow_docs/index.rst

```diff
@@ -10,12 +10,17 @@ Welcome to the NMDC Workflow documentation!
    :caption: Contents:
 
    chapters/overview.rst
-   chapters/1_RQC/index.rst
-   chapters/2_ReadAnalysis/index.rst
-   chapters/3_MetaGAssembly/index.rst
-   chapters/4_MetaGAnnotation/index.rst
-   chapters/5_MAG/index.rst
-   chapters/6_MetaT/index.rst
-   chapters/7_Metaproteomics/index.rst
-   chapters/8_Metabolomics/index.rst
-   chapters/9_NOM/index.rst
+   chapters/1_Metagenome Workflow Overview/index.rst
+   chapters/2_Read Based Taxonomy/index.rst
+   chapters/3_Metagenome Reads QC/index.rst
+   chapters/4_Metagenome Assembly/index.rst
+   chapters/5_Metagenome and Metatranscriptome Annotation/index.rst
+   chapters/6_Metagenome Assembled Genome/index.rst
+   chapters/7_Metatranscriptome Worflow Overview/index.rst
+   chapters/8_Metranscriptome Reads QC/index.rst
+   chapters/9_Metranscriptome Assembly/index.rst
+   chapters/10_Metranscriptome Expression/index.rst
+   chapters/11_Metaproteomics/index.rst
+   chapters/12_Metabolomics/index.rst
+   chapters/13_Natural Organic Matter/index.rst
```

Comment:
FYI: If you end up adding underscores to in `mv` commands (see other comment), these lines will require corresponding changes to match.

Reply:
@eecavanna no spaces or just no spaces b/w the chapter number and chapter title?

Reply:
No spaces anywhere in the string—for example:

- Harder to work with:
  ```
  chapters/1_Metagenome Workflow Overview/index.rst
  ```
- Easier to work with:
  ```
    chapters/1_Metagenome_Workflow_Overview/index.rst
  ```

Reply:
The strings that are in this file as of [the latest commit](https://github.com/microbiomedata/docs/pull/62/commits/c47d56ffda58dd3254ecff12b6cb587ef79893a3) look good to me. Those are:

```
/tmp/book/src/chapters/2_Read_Based_Taxonomy 
/tmp/book/src/chapters/3_Metagenome_Reads_QC 
/tmp/book/src/chapters/4_Metagenome_Assembly
/tmp/book/src/chapters/5_Metagenome_and_Metatranscriptome_Annotatio
/tmp/book/src/chapters/6_Metagenome_Assembled_Genome 
/tmp/book/src/chapters/7_Metatranscriptome_Workflow_Overview
/tmp/book/src/chapters/8_Metatranscriptome_Reads_QC
/tmp/book/src/chapters/9_Metranscriptome_Assembly
/tmp/book/src/chapters/10_Metranscriptome_Expression
/tmp/book/src/chapters/11_Metaproteomics
/tmp/book/src/chapters/12_Metabolomics
/tmp/book/src/chapters/13_Natural_Organic_Matter
```

Reply:
I'm going to add a commit here to make this `src/workflow_docs/index.rst` file consistent with that `src/workflow_docs/fetch_docs_sources.sh` one.

Reply:
Done via commit https://github.com/microbiomedata/docs/pull/62/commits/5070bb075faec18db0444a17c7ff3eca257d797e — resolving thread.